### PR TITLE
Add System module with CPU and Memory stats

### PR DESCRIPTION
**Pull request by ruflin**
* Add note what kind of refactoring is needed for fields.yml collection

This is the first step of https://github.com/elastic/beats/pull/1081

### CODE REVIEW

File: metricbeat/etc/fields.yml [hunk 1]
```diff
@@ -344,3 +344,142 @@ redis:
...
                 User CPU consumed by the background processes
+system:
+  type: group
+  description: >
+    CPU stats for the host machine
+  fields:
+    - name: system-cpu
+      type: group
+      description: >
+        `system-cpu` contains local cpu stats
```
Comment (andrewkroh) [inline metricbeat/etc/fields.yml]: Needs period.

File: metricbeat/etc/fields.yml [hunk 2]
```diff
@@ -344,3 +344,142 @@ redis:
               type: float
               description: >
                 User CPU consumed by the background processes
+system:
+  type: group
+  description: >
+    CPU stats for the host machine
```
Comment (andrewkroh) [inline metricbeat/etc/fields.yml]: It has memory too. How about something like "System status metrics, like CPU and memory usage, collected from the operating system."

File: metricbeat/etc/fields_base.yml [hunk 3]
```diff
@@ -12,6 +12,7 @@ sections:
   - ["apache", "Apache Status"]
   - ["redis", "Redis Info"]
   - ["mysql", "MySQL Status"]
+  - ["system", "System Status"]
```
Comment (andrewkroh) [inline metricbeat/etc/fields_base.yml]: It needs `make fields` to add this to `fields.yml`.

File: metricbeat/etc/fields.yml (hunk 1 above)
Comment (ruflin) [inline metricbeat/etc/fields.yml]: fixed

File: metricbeat/etc/fields_base.yml (hunk 3 above)
Comment (ruflin) [inline metricbeat/etc/fields_base.yml]: Updated

**Comment by ruflin** [comment]
@andrewkroh I suggest to merge this PR as it is now even though the template is actually not correct, as it only takes the first part of the system module (only 1 metricset). My current idea to fix this is introducing a fields.yml on the module level that contains the descriptions for the module. This allows that modules stay self contained. The collection of all fields would then look like: `metricset/fields.yml > module/fields.yml > global/fields.yml` The problem here is that the we need to indent metricset/fields.yml correctly. I was thinking of doing this in python, do you see a simple shell solution?

**Comment by andrewkroh** [comment]
Probably a python script would be simplest to write and understand.

**Pull request merged**